code/game/objects/items/storage/boxes/security_boxes.dm

```diff
@@ -0,0 +1,297 @@
+	illustration = "firingpin"
+
+/obj/item/storage/box/lasertagpins/PopulateContents()
+	for(var/i in 1 to 3)
+		new /obj/item/firing_pin/tag/red(src)
+		new /obj/item/firing_pin/tag/blue(src)
+
+/obj/item/storage/box/handcuffs
+	name = "box of spare handcuffs"
+	desc = "A box full of handcuffs."
+	icon_state = "secbox"
+	illustration = "handcuff"
+
+/obj/item/storage/box/handcuffs/PopulateContents()
+	for(var/i in 1 to 7)
+		new /obj/item/restraints/handcuffs(src)
+
+/obj/item/storage/box/zipties
+	name = "box of spare zipties"
+	desc = "A box full of zipties."
+	icon_state = "secbox"
+	illustration = "handcuff"
+
+/obj/item/storage/box/zipties/PopulateContents()
+	for(var/i in 1 to 7)
+		new /obj/item/restraints/handcuffs/cable/zipties(src)
+
+/obj/item/storage/box/alienhandcuffs
+	name = "box of spare handcuffs"
+	desc = "A box full of handcuffs."
+	icon_state = "alienbox"
+	illustration = "handcuff"
+
+/obj/item/storage/box/alienhandcuffs/PopulateContents()
+	for(var/i in 1 to 7)
+		new /obj/item/restraints/handcuffs/alien(src)
+
+/obj/item/storage/box/rubbershot
+	name = "box of rubber shots"
+	desc = "A box full of rubber shots, designed for riot shotguns."
+	icon_state = "rubbershot_box"
+	illustration = null
+
+/obj/item/storage/box/rubbershot/PopulateContents()
+	for(var/i in 1 to 7)
+		new /obj/item/ammo_casing/shotgun/rubbershot(src)
+
+/obj/item/storage/box/lethalshot
+	name = "box of lethal shotgun shots"
+	desc = "A box full of lethal shots, designed for riot shotguns."
+	icon_state = "lethalshot_box"
+	illustration = null
+
+/obj/item/storage/box/lethalshot/PopulateContents()
+	for(var/i in 1 to 7)
+		new /obj/item/ammo_casing/shotgun/buckshot(src)
+
+/obj/item/storage/box/beanbag
+	name = "box of beanbags"
+	desc = "A box full of beanbag shells."
+	icon_state = "beanbagshot_box"
+	illustration = null
+
+/obj/item/storage/box/beanbag/PopulateContents()
+	for(var/i in 1 to 6)
+		new /obj/item/ammo_casing/shotgun/beanbag(src)
+
+/obj/item/storage/box/emptysandbags
+	name = "box of empty sandbags"
+	illustration = "sandbag"
+
+/obj/item/storage/box/emptysandbags/PopulateContents()
+	for(var/i in 1 to 7)
+		new /obj/item/emptysandbag(src)
+
+/obj/item/storage/box/holy_grenades
+	name = "box of holy hand grenades"
+	desc = "Contains several grenades used to rapidly purge heresy."
+	illustration = "grenade"
+
+/obj/item/storage/box/holy_grenades/PopulateContents()
+	for(var/i in 1 to 7)
+		new/obj/item/grenade/chem_grenade/holy(src)
+
+/obj/item/storage/box/fireworks
+	name = "box of fireworks"
+	desc = "Contains an assortment of fireworks."
+	illustration = "sparkler"
+
+/obj/item/storage/box/fireworks/PopulateContents()
+	for(var/i in 1 to 3)
+		new/obj/item/sparkler(src)
+		new/obj/item/grenade/firecracker(src)
+	new /obj/item/toy/snappop(src)
+
+/obj/item/storage/box/fireworks/dangerous
```

Comment:
You don’t actually need this, just having the proc creates the subtype

Reply:
i have a better idea for this actually